Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst

```diff
@@ -74,6 +83,8 @@ Adding a file
 This example adds a new file in the root folder of the default
 storage::
 
+.. code-block:: php
```

Comment:
This should not be necessary since the default (see `Includes.rst.txt`) already configure PHP highlighting.

Reply:
I have now removed all lines with 

    code-block:: php